pages/tools/fileconverter.vue

```diff
@@ -241,70 +234,44 @@ export default {
           href: 'https://github.com/Geode-solutions/OpenGeode',
         },
       ],
+      loading: false,
+      multiple: false,
+      versions: [],
+      multiple: false,
+      objects: [],
+      success: false,
     }
   },
   computed: {
-    path() {
-      return this.API + '/' + this.ID
-    },
+    ...mapState(['ID', 'cloudRunning']),
   },
   created() {
-    this.CheckID() // Lauches the AWS Lambda function
+    if (process.client) {
+      this.CheckID()
+    }
   },
-  mounted() {},
-  methods: {
-    CheckID() {
-      if (process.client) {
-        console.log(this.$config.API_URL)
-        var ID = localStorage.getItem('ID')
-        if (ID === null) {
-          this.CreateBackEnd()
-        } else {
-          this.ID = ID
-          this.$axios
-            .post(`${this.path}/ping`)
-            .then((response) => {
-              if (response.status == 200) {
-                this.cloudRunning = true
-                this.GetAllowedFiles()
-                this.GetPackagesVersions()
-                this.PingTask()
-              } else {
-                this.CreateBackEnd()
-              }
-            })
-            .catch(() => {
-              this.CreateBackEnd()
-            })
-        }
+  watch: {
+    cloudRunning(newValue) {
+      // console.log(`Updating from ${oldValue} to ${newValue}`)
+
+      // Do whatever makes sense now
+      if (newValue === true) {
+        this.GetAllowedFiles()
+        this.GetPackagesVersions()
       }
     },
-    async CreateBackEnd() {
-      console.log(this.API)
-      await this.$axios
-        .post(`${this.API}/tools/createbackend`)
-        .then((response) => {
-          console.log('response : ', response)
-          if (response.status == 200) {
-            this.ID = response.data.ID
-            localStorage.setItem('ID', this.ID)
-            this.cloudRunning = true
-          } else {
-            console.log('Task creation failed !')
-            this.CreateBackEnd()
-          }
-        })
-      this.GetAllowedFiles()
-      this.GetPackagesVersions()
-      this.PingTask()
-    },
+  },
+  mounted() {},
+  methods: {
+    ...mapActions(['CheckID', 'CreateBackEnd', 'PingTask']),
```

Comment:
Is PingTask still useful in this component?

Reply:
Not anymore